src/EventStore.Core/ClusterVNodeOptions.Framework.cs

```diff
@@ -60,6 +60,34 @@ where defaultValues.TryGetValue(option.Name, out var defaultValue)
 			return builder.Length != 0 ? builder.ToString() : null;
 		}
 
+		public string? CheckForEnvironmentOnlyOptions() {
+			
+			if (ConfigurationRoot != null) {
+				var environmentOptionsOnly = OptionSections.SelectMany(section => section.GetProperties())
+					.Where(option => option.GetCustomAttribute<EnvironmentOnlyAttribute>() != null)
+					.Select(option => option)
+					.ToArray();
+
+				var errorBuilder = new StringBuilder();
+
+				foreach (var provider in ConfigurationRoot.Providers) {
+					var source = provider.GetType();
+
+					if (source.Name is "Default" or "EnvironmentVariables") continue;
```

Comment:
This will fail if the underlying type gets renamed, maybe better to check the type?

Reply:
my 2 cents: 
- Never show a password, by definition 

- that setting shouldn't printed out as the other one since it's _only_ throught environment variable and that might confuse people if  `-DefaultAdminPassword` is printed out as the rest of the settings are 

so --help should really heve another section 
"ENV varaible only" 
 `EVENTSTORE_DEFAULT_ADMIN_PASSWORD` `sets the password of the admin account on first start of the DB`
 
as for the default value: we don't advertise them today in --help, so I would not change anything there 

Question : is the value printed in the dump of the config  at startup in the log ?